From b5a2cd6c7a97428a56562cd5c4c9bf0907ba233f Mon Sep 17 00:00:00 2001
From: Pratham Malaviya <prathamm@mechasystems.com>
Date: Mon, 8 Sep 2025 20:19:52 +0530
Subject: [PATCH] usb: typec: Makefile added tps25751 driver

This patch adds the Makefile entry for TPS25751
PD driver

Signed-off-by: Pratham Malaviya <prathamm@mechasystems.com>

diff --git a/drivers/usb/typec/Makefile b/drivers/usb/typec/Makefile
index 1132a557667b..4b760ee56610 100644
--- a/drivers/usb/typec/Makefile
+++ b/drivers/usb/typec/Makefile
@@ -13,3 +13,4 @@ obj-$(CONFIG_TYPEC_RT1719)	+= rt1719.o
 obj-$(CONFIG_TYPEC_WUSB3801)	+= wusb3801.o
 obj-$(CONFIG_TYPEC)		+= mux/
 obj-$(CONFIG_TYPEC_TPS25750)	+= tps25750.o
+obj-y				+= tps25751.o
-- 
2.25.1

